packages/platforms/accton/x86-64/wedge100bf-32x/onlp/builds/x86_64_accton_wedge100bf_32x/module/src/platform_lib.c

```diff
@@ -31,9 +32,9 @@
 
 #define TTY_DEVICE                      "/dev/ttyACM0"
 #define TTY_PROMPT                      "@bmc:"
-#define TTY_I2C_TIMEOUT                 55000
-#define TTY_BMC_LOGIN_TIMEOUT            1000000
-#define TTY_RETRY                       10
+#define TTY_I2C_TIMEOUT                 80000
+#define TTY_BMC_LOGIN_TIMEOUT           1000000
+#define TTY_RETRY                       20
```

Comment:
How did you determine that 20 is better than 10?

Reply:
Wait time and retry has no document for them. 
Because we get data by log into local tty to BMC, these are variables upon how fast the console responses.

They are both experimental result.  
We accept them for overnight busy run and no error message at /var/log/onlpd.log.

Reply:
It’d be great if you mentioned that in the commit message.